test/rules/src/https_everywhere_checker/rules.py

```diff
@@ -177,6 +183,14 @@ def _determineTestApplication(self):
                  off that rule or exclusion.  Return any coverage problems."""
         if not self.determine_test_application_run:
             self.test_application_problems = []
+
+            # check for duplicated test urls
+            if len(self.tests) != len(set(self.tests)):
+                for test in self.tests:
```

Comment:
Currently, if you run this test it will print one error message for each time the duplicate test URL appears.  You can avoid this by just making this line `for test in set(self.tests):`